gpt_bpe.go

```diff
@@ -102,6 +104,7 @@ const VOCAB_ID_PILE = "pile-tokenizer"
 const VOCAB_ID_CLIP = "clip-tokenizer"
 const VOCAB_ID_NERDSTASH_V1 = "nerdstash_v1-tokenizer"
 const VOCAB_ID_NERDSTASH_V2 = "nerdstash_v2-tokenizer"
+const VOCAB_ID_LLAMA_TWO = "llama_two-tokenizer"
```

Comment:
why `llama_two-tokenizer` insead of `llama2-tokenizer`? How do people normally refer to it?

Reply:
Apparently the Llama 2 and Llama 1 tokenizers are the same. IMO, `llama-tokenizer` would be better.

Reply:
i was unsure about this one, since I don't have the original copy of either, i tried to find a model from each group and tried them and they were *slightly* different, though now that I think of it, it might be differences in the downstream model. I'll rename it for now.